### PR TITLE
Add: Test that modules can define empty lists

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by nickanderson** [comment]
trigger build

**Comment by kacfengine** [comment]
trigger build

File: tests/acceptance/08_commands/01_modules/module_allows_empty_lists.cf [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent test
+{
+  meta:
+      "test_suppress_fail" string => "any",
```
Comment (kacfengine) [inline tests/acceptance/08_commands/01_modules/module_allows_empty_lists.cf]: Should use `test_soft_fail`, otherwise all our builds turn yellow. `test_suppress_fail` are for regressions and more serious bugs that we need to fix very fast, but still don't want to block the build.

I fixed this before merging.

**Comment by kacfengine** [comment]
Merged manually.
